Doc/library/locale.rst

```diff
@@ -366,11 +369,15 @@ The :mod:`locale` module defines the following exception and functions:
    For whole format strings, use :func:`format_string`.
 
 
-.. function:: format_string(format, val, grouping=False)
+.. function:: format_string(format, val, grouping=False, monetary=False)
 
    Processes formatting specifiers as in ``format % val``, but takes the current
    locale settings into account.
 
+   .. versionchanged:: 3.7
+   The *monetary* keyword parameter was added.
+   Replaces :meth:`format`.
```

Comment:
I don't think the 'replaces' sentence is needed.  The deprecated on format covers that.